XenAdmin/Core/Updates.cs

```diff
@@ -472,6 +478,9 @@ private static bool PatchCanBeInstalledOnHost(XenServerPatch serverPatch, Host h
             var appliedUpdates = host.AppliedUpdates();
             var appliedPatches = host.AppliedPatches();
 
+            if (Helpers.productVersionCompare(version.Version.ToString(), host.ProductVersion()) != 0)

```

Comment:
This can be moved above the line 477 (so before `bool elyOrGreater = Helpers.ElyOrGreater(host);`)